starter_code/src/movies.js

```diff
@@ -1,20 +1,53 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
-
+  function turnHoursToMinutes (array) {
+    var moviesModified = array.map(function(e, i) {
+        var split = e.duration.split(" ");
+        var timeInMinutes = 0;
+        if (split.length === 1) {
+          if (split[0].includes("h")){
+              timeInMinutes += parseInt(split[0].replace("h", "")) * 60;
+            } else if (split[0].includes("min")) {
+              timeInMinutes += parseInt(split[0].replace("min", ""))
+            }
+        } else {
+            timeInMinutes += parseInt(split[0].replace("h", "")) * 60;
+            timeInMinutes += parseInt(split[1].replace("min", ""));
+            timeInMinutes += e.duration[3] = " mins";
+        }
+        
+        return Object.assign({}, e, {duration: timeInMinutes} );
```

Comment:
La variable timeInMinutes es de tipo string por tanto el test de Jasmine no os lo paso, si utilizáis parseInt este error se soluciona, además si leemos la documentación del parseInt nos damos cuenta que transforma los primeros números que encuentra en number, por tanto si tenemos un string `"34 mins"` y utilizamos un parseInt lo transformara en `34` de tipo number.
La primera parte estaría perfecta si añadir el parseInt.
`return Object.assign({}, e, {duration: parseInt(timeInMinutes)} );`